features/admin/redirecting_to_previous_filtered_page.feature

```diff
@@ -0,0 +1,38 @@
+@admin_panel
+Feature: Redirecting to previous filtered page
```

Comment:
Cover also a scenario for redirecting to the previous page, when there is more than 1 page and the user was on some other page than the first one